solidity/test/BancorNetwork.js

```diff
@@ -22,9 +22,13 @@ const ConverterV28OrHigherWithoutFallback = contract.fromArtifact('ConverterV28O
 const ConverterV28OrHigherWithFallback = contract.fromArtifact('ConverterV28OrHigherWithFallback');
 
 const LiquidityPoolV1Converter = contract.fromArtifact('LiquidityPoolV1Converter');
+const NetworkSettings = contract.fromArtifact('NetworkSettings');
 
 const DSToken = contract.fromArtifact('DSToken');
 
+const NETWORK_FEE_WALLET = '0x'.padEnd(42, '1');
```

Comment:
Please use a "real" address here or even better - an instance of `TokenHolder` (following my previous PR). In any case, the network fee isn't a constant - it's a "real" address, so using a constant makes the tests confusing to real

Reply:
Fixed.